shuup/admin/modules/currencies/views/list.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.admin.utils.picotable import Column, TextFilter
+from shuup.admin.utils.views import PicotableListView
+from shuup.core.models import Currency
+
+
+class CurrencyListView(PicotableListView):
+    model = Currency
+
+    default_columns = [
+        Column(
+            "code", _(u"Code"), sort_field="code",
+            filter_config=TextFilter(
+                filter_field="code",
+                placeholder=_("Filter by code"),
+            )
+        ),
+        Column(
+            "name", _("Name"), sort_field="translations__name",
```

Comment:
Use babel name here instead of name from he model. You can use the `display` parameter.